scripts/port_forwarding_loop.sh

```diff
@@ -3,9 +3,9 @@
 source scripts/utils.sh
 
 function create_port_forwarding() {
-  kubectl --kubeconfig=${KUBECONFIG} port-forward deployment/$3 $1:$2 --address 0.0.0.0
+  kubectl --kubeconfig=${KUBECONFIG} port-forward deployment/$3 $1:$2 --address 0.0.0.0 -n assisted-installer
```

Comment:
this requires ui deployment fix, it must be deployed to the same namespace

Reply:
i think u can create env NAMESPACE="-n assisted-installer" and use it as ${NAMESPACE}

Reply:
https://issues.redhat.com/browse/MGMT-761